third_party/musl/src/starboard/time/time.c

```diff
@@ -1,11 +1,18 @@
-#include <time.h>
-
 #if SB_API_VERSION < 16
 
-#include "starboard/client_porting/eztime/eztime.h"
+#include <time.h>
+
+#include "starboard/time.h"
 
 time_t time(time_t *t) {
-  return EzTimeTGetNow(t);
+  int64_t posix_us = SbTimeToPosix(SbTimeGetNow());
+  int64_t posix_s = posix_us >= 0 ? posix_us / 1000000
+                                  : (posix_us - 1000000 + 1) / 1000000;
```

Comment:
(not important) no grouping indicators for the zeros here like in `posix_time_test.cc`?

also I haven't quite figured out the arithmetic …